### ISSUE TITLE
aws-ecr-login v1beta1.ParamValue: impossible ParamValues.Type: ""

### ISSUE THREAD

**Issue by tirelibirefe** [tektoncd/pipeline#7267]
/delete

**Comment by v1nc3nt27** [comment]
May I ask what the issue was? I am having the same.

**Comment by vdemeester** [comment]
I'll close because I have no idea what this is… @v1nc3nt27, can you create a new issue with a report of your problem ? (tekton and kubernetes version, status of the `tekton-pipelines` deployments, …)

**Comment by v1nc3nt27** [comment]
It is a duplicate of https://github.com/tektoncd/pipeline/issues/7275. 

I got the error described in the header when I tried to create a Pipeline with results looking like

```
  results:
    - name: password
```

But even adding a type and/or value to the field didnt help. I ended up deleting the results key altogether as it turned our I dont need it anymore for my use case. I have no solution to the actual problem. I upgraded from v0.59.0 to 0.72.0 directly and dont know what changed caused my deployment not needing this key anymore.
